Exiled.CustomModules/API/Features/RespawnManager.cs

```diff
@@ -174,21 +174,24 @@ private void OnPreRespawningTeam(PreRespawningTeamEventArgs ev)
         {
             PreviousKnownTeam = NextKnownTeam;
 
+            // @Nao are you sur that is "or SpawnableTeamType" and not "or not SpawnableTeamType"
```

Comment:
I'm sure

Reply:
Is it not better to do `is not uint` to avoid [casting exeption](https://github.com/warquys/EXILED/blob/1841bbb35953e5a5e02138017317d1a413d5cb34/Exiled.CustomModules/API/Features/RespawnManager.cs#L181) ?

Reply:
Nah, we're kinda safe on it.